lib/utils/keyCommon.js

```diff
@@ -17,21 +17,32 @@ export default class KeyCommon {
   /**
    * @return  {Object} Returns an Key Pair Object with Private and Public keys
    */
-  static createKeyPair() {
-    const privateKeyBytes = secureRandom.randomUint8Array(32);
+  static createKeyPair({rawPrivateKey = ''} = {}) {
```

Comment:
Just one comment, why an object if it's just a parameter ?

Reply:
@betoxd40 Good point! I initially had it as a single string value, but after looking at the other methods, I see that there are other examples of passing in an object even when there is a single parameter, which I believe was done for a) consistency and b) to make it cleaner in the (admittedly unlikely) event we ever wanted to add more params for enhanced functionality. 

Does that make sense?

Reply:
Makes sense for me, thanks!